specification/schemas/Action.json

```diff
@@ -0,0 +1,33 @@
+{
+  "type": "object",
+  "required": [
+    "action_type",
+    "values"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "action_type": {
+      "type": "string",
+      "example": "update-query"
+    },
+    "resource_type": {
+      "type": "string",
+      "example": "services"
+    },
+    "values": {
+      "type": "array",
+      "items": {
+        "$ref": "#/components/schemas/PredicateValue"
```

Comment:
In my proposal these weren't just values, but (I called the database table) `path_values`. Because they didn't just represent values, but values assigned to a specific path in the context of the resource.